docs/_plugins/importMap.cjs

```diff
@@ -77,6 +77,8 @@ async function getCachedImportMap({
 
       generator.importMap.set('@rhds/elements/lib/', '/assets/packages/@rhds/elements/lib/');
 
+      generator.importMap.set('@rhds/tokens/js/color.js', '/assets/packages/@rhds/tokens/js/color.js');
```

Comment:
See comments on other pr

Reply:
will revert.

Reply:
Now that split-demos PR is merged this problem is showing itself again, un-reverting the `/js/` path does seems to work although may not be the correct solution.  PTAL @bennypowers

Reply:
https://github.com/RedHat-UX/red-hat-design-system/pull/1308 fixes this